test/library/standard/IO/useFormattedIO.good

```diff
@@ -1 +1 @@
-useFormattedIO.chpl:1: error: Cannot find module or enum 'FormattedIO'
+useFormattedIO.chpl:1: error: cannot find module or enum named 'FormattedIO'
```

Comment:
How do we decide when to use capitals vs. not?  (e.g., some of the most recent previous instances of "Cannot" at the beginning of a message were capitalized.

Reply:
I think the policy has been to make all brief errors lowercase, and all detailed errors uppercase. Thus, any Dyno-reported brief errors should be lowercase, __unless they are printed verbatim from deprecation warnings or unstable warnings__.

Reply:
Sorry, I think I must've been reading too fast.  I was thinking I'd seen capital-C "Cannots" in earlier .good files, but checking again, I think they were only in the old output, not the new, so the inconsistency I thought I was seeing wasn't there.  My bad.